app/utils/blast-undo-stack.ts

```diff
@@ -0,0 +1,15 @@
+export function blastUndoStackToRemovePasswordFromMemory(
+  element: HTMLInputElement | null,
+  pwLength: number
+) {
+  //
+  // Blast the undo stack a https://bugs.chromium.org/p/chromium/issues/detail?id=961494
+  if (element === null) return;
+  element.focus();
+  // Undo password input, kill undo stack
+  for (let i = 0; i < 256; i++) document.execCommand('undo');
+  // Set value of input same length as password to avoid visual glitch
```

Comment:
Is the `pwLength` arg needed, as opposed to `const newFakeValue = '0'.repeat(element.value.length)`?